pkg/accountmanager/manager_test.go

```diff
@@ -0,0 +1,193 @@
+)
+
+var _ = Describe("Account Manager", func() {
+	Context("Account manager workflow", func() {
+		var (
+			credentials               = "credentials"
+			testAccountNamespacedName = types.NamespacedName{Namespace: "namespace01", Name: "account01"}
+			testSecretNamespacedName  = types.NamespacedName{Namespace: "namespace01", Name: "secret01"}
+			testCesNamespacedName     = types.NamespacedName{Namespace: "namespace01", Name: "Ces01"}
+			account                   *v1alpha1.CloudProviderAccount
+			accountManager            *AccountManager
+			fakeClient                client.WithWatch
+			secret                    *corev1.Secret
+			ces                       *v1alpha1.CloudEntitySelector
+			accountCloudType          runtimev1alpha1.CloudProvider
+			pollIntv                  uint
+		)
+
+		BeforeEach(func() {
+
+			var err error
+			newScheme := runtime.NewScheme()
+			utilruntime.Must(clientgoscheme.AddToScheme(newScheme))
+			utilruntime.Must(v1alpha1.AddToScheme(newScheme))
+
+			fakeClient = fake.NewClientBuilder().WithScheme(newScheme).Build()
+			cloudInventory := inventory.InitInventory()
+			accountManager = &AccountManager{
+				Log:       logf.Log,
+				Client:    fakeClient,
+				Inventory: cloudInventory,
+				mutex:     sync.RWMutex{},
+			}
+
+			pollIntv = 1
+			account = &v1alpha1.CloudProviderAccount{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      testAccountNamespacedName.Name,
+					Namespace: testAccountNamespacedName.Namespace,
+				},
+				Spec: v1alpha1.CloudProviderAccountSpec{
+					PollIntervalInSeconds: &pollIntv,
+					AWSConfig: &v1alpha1.CloudProviderAccountAWSConfig{
+						Region: "us-east-1",
+						SecretRef: &v1alpha1.SecretReference{
+							Name:      testSecretNamespacedName.Name,
+							Namespace: testSecretNamespacedName.Namespace,
+							Key:       credentials,
+						},
+					},
+				},
+			}
+			credential := `{"accessKeyId": "keyId","accessKeySecret": "keySecret"}`
+			secret = &corev1.Secret{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      testSecretNamespacedName.Name,
+					Namespace: testSecretNamespacedName.Namespace,
+				},
+				Data: map[string][]byte{
+					"credentials": []byte(credential),
+				},
+			}
+			ces = &v1alpha1.CloudEntitySelector{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      testCesNamespacedName.Name,
+					Namespace: testCesNamespacedName.Namespace,
+					OwnerReferences: []v1.OwnerReference{
+						{
+							APIVersion: "crd.cloud.antrea.io/crdv1alpha1",
+							Kind:       "CloudProviderAccount",
+							Name:       testAccountNamespacedName.Name,
+						},
+					},
+				},
+				Spec: v1alpha1.CloudEntitySelectorSpec{
+					AccountName: testAccountNamespacedName.Name,
+					VMSelector: []v1alpha1.VirtualMachineSelector{
+						{
+							VpcMatch: &v1alpha1.EntityMatch{
+								MatchID: "xyzq",
+							},
+						},
+					},
+				},
+			}
+			accountManager.ConfigureAccountManager()
+			accountCloudType, err = util.GetAccountProviderType(account)
```

Comment:
Remove L44
```suggestion
			accountCloudType, err := util.GetAccountProviderType(account)
```

Reply:
accountCloudType is already defined. So we also should declare err as well.